cloudfoundry/resource_cf_app.go

```diff
@@ -87,6 +87,12 @@ func resourceApp() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"buildpacks": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
```

Comment:
```suggestion
				Elem:     &schema.Schema{Type: schema.TypeString},
				ConflictsWith: []string{"buildpack"},
```

Reply:
Also I think the same is needed for "buildpack" attribute

Reply:
Added ConflictsWith to both buildpack and buildpacks